proteus-broker-mgmt-idl/src/main/proto/proteus/cluster_mgmt.proto

```diff
@@ -0,0 +1,33 @@
+//
+//    Copyright 2019 The Proteus Authors
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+syntax = "proto3";
+
+package io.netifi.proteus.broker.info;
+
+import "google/protobuf/empty.proto";
+import "proteus/broker_mgmt.proto";
+import "proteus/broker_info.proto";
+
+option java_package = "io.netifi.proteus.cluster.mgmt";
+option java_outer_classname = "ClusterManagement";
+option java_multiple_files = true;
+
+service ClusterManagementService {
+    // Closes connections to a specific set of destinations across broker cluster
+    rpc closeDestination (stream io.netifi.proteus.broker.info.Destination) returns (Ack) {}
+
+    // Closes connections to a specific set of groups across broker cluster
```

Comment:
nit: Closes connections to a specific set of groups across broker cluster
Probably should be: Closes all connections to a specific group cluster-wide